internal/cache/db_test.go

```diff
@@ -510,7 +510,7 @@ func initCache(t *testing.T, dbFile string) (c *cache.Cache, cacheDir string) {
 		createDBFile(t, filepath.Join("testdata", dbFile+".db.yaml"), cacheDir)
 	}
 
-	c, err := cache.New(cacheDir)
+	c, err := cache.New(cacheDir, cache.WithoutCleaningOnNew())
```

Comment:
I'm pretty sure this is not the best solution for the "noise" problem, but I wasn't sure what was the best way to handle the amount of diff that would be generated when updating most of the test cases and fixtures.

@didrocks, any insights on this? Maybe it should be tackled on a separate `tests(cache)` PR after this one gets merged? I'll keep it as a draft until we decide how to handle this.

Reply:
Maybe instead of this fixture, we can have an exported_tests on cache which set a user last login back to the path? That way, we open the cache, all is good, and we reset the time, and test triggering the cleaning, wdyt?

Reply:
I'll try this out. Thanks!

Reply:
btw, most of the tests should not trigger cleaning, hence a With function by default in the tests to put the cleaning time before 2004.